src/leap/bitmask/backend/backend_proxy.py

```diff
@@ -157,6 +225,8 @@ def _api_call(self, *args, **kwargs):
             'arguments': kwargs,
         }
 
+        request_json = None
```

Comment:
Not needed indeed, but maybe good practice to ensure the variable exists in enclosing context? Couldn't find a reference, though.